Neos.Media/Classes/Domain/Model/Adjustment/ResizeImageAdjustment.php

```diff
@@ -294,8 +300,7 @@ public function canBeApplied(ImagineImageInterface $image)
      */
     public function applyToImage(ImagineImageInterface $image)
     {
-        $ratioMode = $this->ratioMode ?: ImageInterface::RATIOMODE_INSET;
```

Comment:
@kdambekalns currently I see only empty strings stored as `ratioMode` for many images. Your migration only changes "null" => "inset", but leaves these untouched.

Not sure why this is so, but I have never seen an entry in the database where one of the two possible strings are set. So if you remove this check here and have it fallback to "INSET", you will probably end up with Exceptions like `Invalid mode specified` from the `resize()` method.

An "easy" approach could be to make sure the getter is always used and have the getter apply the proper default if the string is empty.

An additional way would be to also migrate all other records where there is currently an "empty string" stored. But of course this keeps the mystery on how these objects have been created in the first place.

Reply:
> Your migration only changes "null" => "inset", but leaves these untouched

The mode was always set to `inset` in the code, when not set in the DB. The default value so far was `null`, and that lead to problems as you pointed out in https://github.com/neos/neos-development-collection/pull/2716/files#r350120975 - the empty strings are probably the result of some type casting during the lifetime of those objects.

> also migrate all other records where there is currently an "empty string" stored

I updated the migration to set the default also for empty strings.